<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NMediaManager::NPreferences::NUi::CFFMpegInfo</class>
 <widget class="QWidget" name="NMediaManager::NPreferences::NUi::CFFMpegInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="containersTab">
      <attribute name="title">
       <string>Containers/Formats</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="supportedFormatsGB">
         <property name="title">
          <string>Containers:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QListWidget" name="supportedFormats">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="videoTab">
      <attribute name="title">
       <string>Video</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QGroupBox" name="videoDecodingCodecsGB">
         <property name="title">
          <string>Video Decoding Codecs:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QListWidget" name="videoDecodingCodecs">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="videoEncodersGB">
         <property name="title">
          <string>Video Encoders:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QListWidget" name="videoEncoders">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="videoDecodersGB">
         <property name="title">
          <string>Video Decoders:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_18">
          <item>
           <widget class="QListWidget" name="videoDecoders">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="videoEncodingCodecsGB">
         <property name="title">
          <string>Video Encoding Codecs:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QListWidget" name="videoEncodingCodecs">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QGroupBox" name="videoDecoderExtensionsGB">
         <property name="title">
          <string>Video Decoder Extensions:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_22">
          <item>
           <widget class="QListWidget" name="videoDecoderExtensions"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="videoEncoderExtensionsGB">
         <property name="title">
          <string>Video Encoder Extensions:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QListWidget" name="videoEncoderExtensions">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="audioTab">
      <attribute name="title">
       <string>Audio</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QGroupBox" name="audioDecodersGB">
         <property name="title">
          <string>Audio Decoders:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QListWidget" name="audioDecoders">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="audioDecodingCodecsGB">
         <property name="title">
          <string>Audio Decoding Codecs:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_19">
          <item>
           <widget class="QListWidget" name="audioDecodingCodecs">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="audioEncodingCodecsGB">
         <property name="title">
          <string>Audio Encoding Codecs:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QListWidget" name="audioEncodingCodecs">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="audioEncodersGB">
         <property name="title">
          <string>Audio Encoders:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QListWidget" name="audioEncoders">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QGroupBox" name="audioDecoderExtensionsGB">
         <property name="title">
          <string>Audio Decoder Extensions:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QListWidget" name="audioDecoderExtensions"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="audioEncoderExtensionsGB">
         <property name="title">
          <string>Audio Encoder Extensions:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QListWidget" name="audioEncoderExtensions">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="subtitleTab">
      <attribute name="title">
       <string>Subtitles</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="subtitleEncodingCodecsGB">
         <property name="title">
          <string>Subtitle Encoding Codecs:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_20">
          <item>
           <widget class="QListWidget" name="subtitleEncodingCodecs">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="subtitleDecodingCodecsGB">
         <property name="title">
          <string>Subtitle Decoding Codecs:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_21">
          <item>
           <widget class="QListWidget" name="subtitleDecodingCodecs">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="subtitleDecodersGB">
         <property name="title">
          <string>Subtitle Decoders:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_16">
          <item>
           <widget class="QListWidget" name="subtitleDecoders">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="subtitleEncodersGB">
         <property name="title">
          <string>Subtitle Encoders:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_15">
          <item>
           <widget class="QListWidget" name="subtitleEncoders">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QGroupBox" name="subtitleDecoderExtensionsGB">
         <property name="title">
          <string>Subtitle Decoder Extensions:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_23">
          <item>
           <widget class="QListWidget" name="subtitleDecoderExtensions"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="subtitleEncoderExtensionsGB">
         <property name="title">
          <string>Subtitle Encoder Extensions:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <widget class="QListWidget" name="subtitleEncoderExtensions">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="hwAccelerators">
      <attribute name="title">
       <string>Hardware Accelerators</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="hwAccelsGB">
         <property name="title">
          <string>Hardware Accelerators</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_17">
          <item>
           <widget class="QListWidget" name="hwAccels"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="recomputeBtn">
       <property name="text">
        <string>Re-compute Defaults</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>227</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>supportedFormats</tabstop>
  <tabstop>videoEncodingCodecs</tabstop>
  <tabstop>videoDecodingCodecs</tabstop>
  <tabstop>videoEncoders</tabstop>
  <tabstop>videoDecoders</tabstop>
  <tabstop>videoEncoderExtensions</tabstop>
  <tabstop>videoDecoderExtensions</tabstop>
  <tabstop>audioEncodingCodecs</tabstop>
  <tabstop>audioDecodingCodecs</tabstop>
  <tabstop>audioEncoders</tabstop>
  <tabstop>audioDecoders</tabstop>
  <tabstop>audioEncoderExtensions</tabstop>
  <tabstop>audioDecoderExtensions</tabstop>
  <tabstop>subtitleEncodingCodecs</tabstop>
  <tabstop>subtitleDecodingCodecs</tabstop>
  <tabstop>subtitleEncoders</tabstop>
  <tabstop>subtitleDecoders</tabstop>
  <tabstop>subtitleEncoderExtensions</tabstop>
  <tabstop>subtitleDecoderExtensions</tabstop>
  <tabstop>hwAccels</tabstop>
  <tabstop>recomputeBtn</tabstop>
 </tabstops>
 <resources>
  <include location="../../SABUtils/resources/SABUtils.qrc"/>
 </resources>
 <connections/>
</ui>
